lib/modules/platform/bitbucket/index.ts

```diff
@@ -623,7 +623,15 @@ export async function addReviewers(
 
   const body = {
     title,
-    reviewers: reviewers.map((username: string) => ({ username })),
+    reviewers: reviewers.map((username: string) => {
+      let key = 'username';
+      if (checkUUID(username)) {
+        key = 'uuid';
+      }
```

Comment:
```suggestion
      const key = isUUID(username) ? 'uuid' : 'username';
```
small simplification